teemaderegister-be/src/controllers/users.js

```diff
@@ -81,13 +84,35 @@ module.exports.updateUser = async (req, res) => {
     })
   if (userWithSameEmail) throw new Error(`Email ${email} already in use`)
 
+  const user = await User.findById(req.user._id);
+
+  //const user = await User.findOne({'login.email': email})
+
+  const newtoken = await crypto.randomBytes(50).toString('hex');
+  let confirmLink = "http://localhost:3000/api/auth/emailconfirm/" + newtoken
```

Comment:
link should be for some frontend URL like for example 

http://localhost:8080/settings/account?emailConfirmToken=.....